mesheryctl/internal/cli/root/model/import.go

```
@@ -69,6 +73,55 @@ mesehryctl model import -f [path-to-models]
 			}
 			return nil
 		}
+
+		hasCSVs, err := hasCSVs(path)
+		if err != nil {
+			utils.Log.Error(err)
+		}
+
+		if hasCSVs {
+			modelcsvpath, componentcsvpath, relationshipcsvpath, err := utils.GetCsv(path)
+			if err != nil {
+				utils.Log.Infof("%s: %s", utils.BoldString("ERROR"), "Error importing model using CSV files")
+				if meshkitErr, ok := err.(*errors.Error); ok {
+					if len(meshkitErr.ProbableCause) != 0 {
+						utils.Log.Infof("\n  %s:\n  %s", utils.BoldString("PROBABLE CAUSE"), strings.Join(meshkitErr.ProbableCause, ". "))
+					}
+					if len(meshkitErr.SuggestedRemediation) != 0 {
+						utils.Log.Infof("\n  %s:\n  %s", utils.BoldString("SUGGESTED REMEDIATION"), strings.Join(meshkitErr.SuggestedRemediation, ". "))
+					}
+				} else {
+					utils.Log.Error(err)
+				}
+
+				return nil
```

we should return the error because cli will exit with 0 instead of 1